src/main/java/org/example/MessageServiceImpl.java

```diff
@@ -3,40 +3,35 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 import java.time.temporal.ChronoUnit;
 
-public class MessageServiceImpl implements MessageService{
+public class MessageServiceImpl implements MessageService {
 
-  private static final LocalDate baselineDate = LocalDate.parse("2012-08-06",
-      DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+  // format for ISO 8601
+  private static final DateTimeFormatter isoFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static final LocalDate curiosityLandingDate = LocalDate.parse("2012-08-06",
+      isoFormat);
 
   @Override
   public String getHello() {
     return "Hello World Jersey and HK2 ";
   }
 
   @Override
-  public int curiosityDateDateConversion(String processingDate) {
-    float diff_In_Days ;
-    LocalDate earthDate;
-    // format for ISO 8601
-    DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  public int  curiosityDateConversion(String date) {
```

Comment:
It might sound pedantic, but double spaces are making code less readable as they break the rhythm. Most teams are quite picky about that (we even have automatic formatting checks blocking pull requests).
```suggestion
  public int curiosityDateConversion(String date) {
```

Reply:
Sure I will follow it.